compose_files/pki/certs/main.conf

```diff
@@ -7,3 +7,6 @@ subjectAltName = @alt_names
 DNS.1 = mike-virtual-machine
 DNS.2 = auth.test
 DNS.3 = cwms-data.test
+DNS.1 = mike-virtual-machine
+DNS.2 = auth.test
+DNS.3 = cwms-data.test
```

Comment:
extra lines seem to keep coming back.
```suggestion

```

Reply:
Hmm, maybe I should just delete and add those to the .gitignore. it is intentionally using the local hostname as one of the SANs. But that is an extra user step.